tests/lib/rules/lines-around-comment.js

```diff
@@ -317,8 +317,195 @@ ruleTester.run("lines-around-comment", rule, {
                 allowBlockEnd: true
             }],
             ecmaFeatures: {classes: true}
-        }
+        },
+
+        // check for object start comments
```

Comment:
> Also can you please split the code into multi-line has its really difficult for the reviewer to read and understand.

I just copied the style that was in that file already. I can change it for multi-line, sure. I also recall that in all my past ESLint contributions the test code was always on a single line like that.